include/libtorrent/buffer.hpp

```diff
@@ -75,7 +68,7 @@ class buffer
 
 		if (size == 0) return;
 
-		size = round_up8(size);
+		size = (size + 7) & (~std::size_t(0x7)); //round_up8(size);
```

Comment:
no, it's not about fewer lines :)  The main reason for this PR is the problematic use of an anonymous namespace in a header file. It's documented to be the cause of bloated final binaries and even ODR violations. I will change the comment with your information.